dynamodb_lock/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "dynamodb_lock"
+description = "Distributed lock backed by Dynamodb"
+version = "0.4.1"
+authors = ["Qingping Hou <qph@scribd.com>"]
```

Comment:
I didn't write this, @mosyp did :D  @mosyp could you help suggest a change for your preferred email here?